src/redirectPage.tsx

```diff
@@ -9,7 +9,7 @@ import { error } from './lib/logger.ts'
 const ConfigIframe = (): JSX.Element => {
   const { parentDomain } = getSubdomainParts(window.location.href)
 
-  const iframeSrc = `${window.location.protocol}//${parentDomain}/config?origin=${encodeURIComponent(window.location.origin)}`
+  const iframeSrc = `${window.location.protocol}//${parentDomain}:${window.location.port}/config?origin=${encodeURIComponent(window.location.origin)}`
```

Comment:
this will fail when no port is specified. i.e. `http://localhost:/config?...`
```suggestion
  const portString = window.location.port === '' ? '' : `:${window.location.port}`
  const iframeSrc = `${window.location.protocol}//${parentDomain}${portString}/config?origin=${encodeURIComponent(window.location.origin)}`
```

Reply:
Nice! Thanks